mne_bids/tests/test_write.py

```diff
@@ -2659,6 +2660,28 @@ def test_anonymize(subject, dir_name, fname, reader, tmp_path):
         assert _raw.info['meas_date'].day == 1
     assert raw2.info['meas_date'].year < 1925
 
+    # write without sources
+    scans_fname = BIDSPath(subject=bids_path.subject,
+                           session=bids_path.session,
+                           suffix='scans', extension='.tsv',
+                           root=bids_path.root)
+    anonymize['keep_source'] = False
+    bids_path = \
+        write_raw_bids(raw, orig_bids_path, overwrite=True,
+                       anonymize=anonymize, verbose=False)
+    scans_tsv = _from_tsv(scans_fname)
+    assert 'sources' not in scans_tsv.keys()
+
+    # Write with sources this time get the scans tsv
+    with pytest.warns(RuntimeWarning, match='`daysback` is too small'):
```

Comment:
Why don't you use a suitable `daysback` for this test?

Reply:
Done.